.travis.yml

```diff
@@ -56,11 +56,9 @@ before_script:
 script:
   - bin/build-image
   - kind load docker-image "cfcontainerization/cf-operator:dev" --name kind$KUBE
-  - bin/test-integration
-  - bin/test-integration storage
-  - bin/test-integration util
-  - bin/test-cli-e2e
-  - bin/test-helm-e2e
+  - make test-integration test-integration-storage test-integration-subcmds
```

Comment:
I'd prefer using the bin scripts directly, in travis there is no need to check for dependencies every time

Reply:
Fixed in https://github.com/cloudfoundry-incubator/quarks-operator/pull/942/commits/99af96c7783f50d1b4d5801e595a96987e956c28 :+1: